### PR TITLE
Update brjs to use service box

### PR DESCRIPTION
**Pull request by germtb**

### CODE REVIEW

File: brjs-sdk/sdk/libs/javascript/br-formatting/tests/test-unit/js-test-driver/tests/DateFormatterTest.js [hunk 1]
```diff
@@ -2,9 +2,11 @@
 	DateFormatterTest = TestCase("DateFormatterTest");
 
 	var DateFormatter = require('br/presenter/formatter/DateFormatter');
+	var ThousandsFormatter = require('br/presenter/formatter/ThousandsFormatter');
 
 	DateFormatterTest.prototype.setUp = function() {
 
+		debugger;
 		this.m_oDate = Date;
```
Comment (Aivarass) [inline brjs-sdk/sdk/libs/javascript/br-formatting/tests/test-unit/js-test-driver/tests/DateFormatterTest.js]: Remove debugger!

File: brjs-sdk/sdk/libs/javascript/browser-modules/browser-modules.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// THIS IS THE ONE
```
Comment (Aivarass) [inline brjs-sdk/sdk/libs/javascript/browser-modules/browser-modules.js]: Weird comment

**Comment by thecapdan** [comment]
this PR was reverted from master and will need to be reopened

File: brjs-sdk/sdk/libs/javascript/br-formatting/tests/test-unit/js-test-driver/tests/DateFormatterTest.js [hunk 3]
```diff
@@ -2,6 +2,7 @@
 	DateFormatterTest = TestCase("DateFormatterTest");
 
 	var DateFormatter = require('br/presenter/formatter/DateFormatter');
+	var ThousandsFormatter = require('br/presenter/formatter/ThousandsFormatter');
```
Comment (dogoku) [inline brjs-sdk/sdk/libs/javascript/br-formatting/tests/test-unit/js-test-driver/tests/DateFormatterTest.js]: is this line really needed in here?

**Pull request merged**